README.md

```diff
@@ -77,6 +77,9 @@ Pypi:
 
 
 ### API Usage
+[example](example/api.py)
```

Comment:
Since the code is already inline, is another file required? Also there is no mention of `requirements.txt`. It is probably better to give instructions to `pip install`.